api/app/serializers/v1/version_serializer.rb

```diff
@@ -3,29 +3,44 @@ class VersionSerializer < ManifoldSerializer
 
     include ::V1::Concerns::ManifoldSerializer
 
-    typed_attribute :item_type, NilClass
-    typed_attribute :item_id, NilClass
-    typed_attribute :event, NilClass
-    typed_attribute :created_at, NilClass
+    typed_attribute :item_type, Types::String.meta(read_only: true)
+    typed_attribute :item_id, Types::Serializer::ID.meta(read_only: true)
+    typed_attribute :event, Types::String.meta(read_only: true)
+    typed_attribute :created_at, Types::DateTime.meta(read_only: true)
 
     # Strip out lateral, nil-nil changes and updated_at
-    typed_attribute :object_changes, Hash do |object|
+    typed_attribute :object_changes, Types::Hash.meta(
+      description: "Each key contains an array of values that describe how that project has changed over time. "\
+      "This could be a history of changes for the project regarding title, description, slug, etc.\n"\
+      'For example: "featured": [ false, true ]\n'\
+      'This means the project was changed from "not featured" to "featured"',
+      read_only: true
+    ) do |object|
       object.object_changes&.except("updated_at")
     end
 
-    typed_attribute :actor_name, NilClass do |object|
+    typed_attribute :actor_name, Types::String.meta(
+      description: "The name of the user that made the change",
```

Comment:
Is this a good description for the actor name?

Reply:
Yep!